presto-tests/src/test/java/com/facebook/presto/tests/TestQueryManager.java

```diff
@@ -237,28 +237,29 @@ public void testQueryOutputSizeExceeded()
         }
     }
 
-    // Flaky test: https://github.com/prestodb/presto/issues/20447
-    @Test(enabled = false)
+    @Test(timeOut = 120_000L)
     public void testQueryCountMetrics()
             throws Exception
     {
         DispatchManager dispatchManager = queryRunner.getCoordinator().getDispatchManager();
-        // Create a total of 10 queries to test concurrency limit and
+        // Create a total of 6 queries to test concurrency limit and
         // ensure that some queries are queued as concurrency limit is 3
-        createQueries(dispatchManager, 10);
+        createQueries(dispatchManager, 6);
```

Comment:
2-3 out of 10 queries were consistently failing for me on my local mac complaining of insufficient resources